kitsune/search/v2/documents.py

```diff
@@ -244,6 +275,10 @@ def prepare_groups(self, instance):
     def get_model(cls):
         return Profile
 
+    @classmethod
```

Comment:
why `classmethod`?

Reply:
I guess the same Q holds true for all the other occurrences

Reply:
It needs to be able to refer to the `get_model` classmethod to get the model and then its manager.